CMakeLists.txt

```diff
@@ -21,16 +21,10 @@
 PROJECT(mudlet)
 CMAKE_MINIMUM_REQUIRED(VERSION 3.1)
```

Comment:
Remind me, why do we now require 3.1 (as I have 2.8.12.2, oh! That was a local build, better uninstall that now, right I have a system provided 3.0.2 to hand in my Debian Stable "Jessie" system)?